.github/workflows/build.yml

```diff
@@ -306,6 +306,55 @@ jobs:
           name: ${{ steps.diff.outputs.diff_name }}
           path: ${{ steps.diff.outputs.diff_name }}
 
+  create-console-preview:
+    name: "Create console preview"
+    runs-on: ubuntu-latest
+    needs:
+      - build
+    env:
+      FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }}
+      FLY_VERSION: "0.1.80"
+      APP_NAME: wing-console-pr-${{github.event.number}}
+    if: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository }}
```

Comment:
This should also ignore merge queue PRs like https://github.com/winglang/wing/pull/4003

(should be able to ignore any branches that start with `mergify/merge-queue/`)